src/deltacode/utils.py

```diff
@@ -182,6 +182,18 @@ def deltas(deltacode, all_delta_types=False):
         elif not delta.is_unmodified():
             yield delta.to_dict()
 
+def stats(deltacode):
```

Comment:
No need for this function if you are just going to run `deltacode.stats.percent_to_dict()`

Reply:
@majurg so from the cli.py file I directly call the `stats.to_dict() `method?

Reply:
yes